packages/@atjson/source-html/src/parser.ts

```diff
@@ -125,20 +125,21 @@ export default class Parser {
     }
 
     if (location.startTag && location.endTag) {
-      let start = this.convertTag(node, "startTag");
+      let start = location.startTag.startOffset - this.offset;
+      this.convertTag(node, "startTag");
```

Comment:
@blaine The changes in this file are perhaps the most significant in this PR, as this changes the behavior of our HTML source to have html annotations wrap around their start/end tags rather than nest inside of them:
```
<b>bold!</b> 
   ^-b-^
```
vs 
```
<b>bold!</b> 
^----b-----^
```

This is a change in our behavior for a format like Markdown. My justification is the previous behavior is more useful for text manipulation, which we might care a lot about when transforming markdown or when writing a text editor, whereas this behavior is more useful for the conversion/manipulation of the elements themselves, which I think will be the primary use of this source. But I'm interested to hear your thoughts.